.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "stable"
+# TODO: switch back to `npm test` default once tests are fixed.
+script: npm run lint
```

Comment:
this was easier for now because we have to figure out phantomjs and all that for current tests. Maybe we can split up tests into real unit tests and ones that require a browser to connect to.